include/npcomp/Dialect/TCF/IR/TCFOps.td

```diff
@@ -95,4 +95,24 @@ def TCF_MatmulOp : TCF_Op<"matmul"> {
   let assemblyFormat = "$lhs `,` $rhs attr-dict `:` functional-type(operands, results)";
 }
 
+def TCF_ConvNCHWOp : TCF_Op<"conv_2d_nchw"> {
+  let summary = "2-D convolution";
+  let description = [{
+    Performs 2-D convolution.
+
+    The tensors have dimensions:
+    - in:     [N, C, H, W]
+    - filter: [F, C, KH, KW]
+    - result: [N, F, H, W]
```

Comment:
this documentation is not correct.

Reply:
Great catch. `result` should refer to  Hout/Wout or something besides H/W, input shape.